plugins/analytics-module-matomo/src/index.ts

```diff
@@ -1,2 +1,16 @@
+import {
+  analyticsApiRef,
+  configApiRef,
+  createApiFactory,
+} from '@backstage/core-plugin-api';
+
+import { MatomoAnalytics } from './api';
+
 export { analyticsModuleMatomoPlugin } from './plugin';
 export * from './api';
+
+export const MatomoAnalyticsApi = createApiFactory({
```

Comment:
LGTM though I'd suggest to keep this instantiation either alongside your plugin declaration in ./plugin.ts or in a file within ./api and just re-export the modules from this file.

Reply:
Makes sense. Updated the commit.